Source/WebCore/rendering/svg/legacy/SVGResourcesCycleSolver.h

```diff
@@ -35,7 +35,6 @@ class SVGResourcesCycleSolver {
     static void resolveCycles(RenderElement&, SVGResources&);
 
 private:
-    SVGResourcesCycleSolver() { }
```

Comment:
Deleting this isn’t right, because then the `SVGResourcesCycleSolver` default constructor will be public instead of private. So please add this back with `= default`.